policyengine_us/parameters/gov/states/ks/tax/income/deductions/standard/extra_amount.yaml

```
@@ -0,0 +1,23 @@
+description: Extra amount a KS income tax unit can get as a standard deduction.
+metadata:
+  label: KS extra standard deduction for elderly and blind
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2021 Form K-40 instructions
+    href: https://www.ksrevenue.gov/pdf/ip21.pdf
+  - title: 2022 Form K-40 instructions
+    href: https://www.ksrevenue.gov/pdf/ip22.pdf
+
+JOINT:
+  2021-01-01: 700
```

and for other statuses per law
```suggestion
  1998-01-01: 700
```

---

Done